auth/credentials/idtoken/compute.go

```diff
@@ -65,6 +68,7 @@ func (c computeIDTokenProvider) Token(ctx context.Context) (*auth.Token, error)
 	if c.format == ComputeTokenFormatFullWithLicense {
 		v.Set("licenses", "TRUE")
 	}
+	callctx.SetHeaders(ctx, header.GOOGLE_API_CLIENT_HEADER, header.GetGoogHeaderToken(header.CredTypeMDS, header.TokenTypeID))
```

Comment:
Should this be commented out as well for now? Also, this returns a context who has that values set -- make sure to reassign.

Reply:
Happens elsewhere too.

Reply:
Fixed, thanks!